primitives/src/v1.rs

```diff
@@ -710,6 +749,9 @@ sp_api::decl_runtime_apis! {
 		/// This can be used to instantiate a `SigningContext`.
 		fn session_index_for_child() -> SessionIndex;
 
+		/// Yields the session info for the given session, if stored.
+		// fn session_info(index: SessionIndex) -> Option<SessionInfo>;
+
```

Comment:
I'll remove this for now.
```suggestion
		/// Yields the session info for the given session, if stored.
		// fn session_info(index: SessionIndex) -> Option<SessionInfo>;

```